app/src/main/kotlin/com/saltedge/authenticator/tools/DateTools.kt

```diff
@@ -35,6 +35,16 @@ import org.joda.time.LocalDate
 fun DateTime.toDateFormatString(appContext: Context): String =
     this.toString("d MMMM yyyy", appContext.getCurrentAppLocale()) ?: ""
 
+/**
+ * Converts the current date time to a string presentation
```

Comment:
Add mention, that this date time string is in UTC format

Reply:
Please see what is written below: 
`@return the date as a string with time and time zone`